src/driver/koa/KoaDriver.ts

```diff
@@ -111,6 +111,9 @@ export class KoaDriver extends BaseDriver implements Driver {
                 .forEach(param => {
                     defaultMiddlewares.push(multer(param.extraOptions).array(param.name));
                 });
+
+            // HACK: should be removed when koa-multer merges #15: https://github.com/koa-modules/multer/pull/15
```

Comment:
No need to add the almost same comment twice, remove the comment here.

Reply:
Done